reduxApp2/dev/js/components/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import FillDetails from '../containers/fill-details';
+import UserDetails from '../containers/user-details';
+require('../../scss/style.scss');
```

Comment:
@VARTIKASINGH1234 why you're using require statement (es5) for importing styles ?

Reply:
done